### ISSUE TITLE
weird patterns on translucent textures

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#68]
This bug affects the for-0.51.0 branch, I cannot reproduce with a 0.50.0 build and I haven't tried master.

Look at these screenshots of the `hangar28` maps by _tvezet_ and the `thunder` one by _EmperorJack_:

[![wrong translucency hangar28](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-09_231104_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-09_231104_000.jpg)
[![wrong translucency thunder](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-10_012544_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-10_012544_000.jpg)

The translucency looks obviously wrong. That's not a compression fault (I have not recompressed the thunder's textures and it's rendered right on 0.50.0 but rendered wrong on for-0.51.0).

These is the two related shaders:

* The one used in `thunder` map, from `map-thunder` package:

```
textures/thunder/nexusglass
{
	qer_editorimage textures/thunder/nexusglass
	qer_trans .40
	surfaceparm trans
	cull none
	{
		map textures/thunder/nexusglass
		blendfunc add
		rgbGen const ( 0.01 0.01 0.01 )
	}
	{
		map $lightmap 
		blendfunc gl_dst_color gl_src_alpha
		rgbGen identity
		tcGen lightmap 
	}
}
```

* The one used in `hangar28` from `tex-trak5` package:

```
textures/shared_trak5/glass
{
	qer_editorImage     textures/shared_trak5_src/glass_p
	qer_trans           0.30

	cull                none
	surfaceparm         trans

	{
		map       textures/shared_trak5_src/glass_d
		stage     diffuseMap
		blend     blend
	}
}
```

**Comment by illwieckz** [comment]
this is very likely to be a lightmap issue

**Comment by illwieckz** [comment]
it's really awful:
[![translucency bug](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013656_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013656_000.jpg)

it really looks to be related to lightmaps,

- with lightmap:  
[![translucency bug](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013731_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013731_000.jpg)
- without lightmap:  
[![translucency bug](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013820_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013820_000.jpg)

**Comment by illwieckz** [comment]
This looks very close to #76 as the same action makes the bug disappear (disabling deluxeMaps).

Note that with a more recent driver the artifacts looks a bit different (pure white instead of pure black, etc.) but this is probably not an important information since unexpected behavior are unexpected. :-)

**Comment by Viech** [comment]
I does indeed look like another problem with deluxemaps. If these take on weird values you'd get weird reflections and weird shadows. I have no idea why this would affect translucent surfaces more than others, though.

**Comment by illwieckz** [comment]
@gimhael do you have any idea about what's happening with deluxemaps?

**Comment by illwieckz** [comment]
The translucent problem in this screenshot is related to this issue fixed by #87 PR, the white stuff is another issue and probably related to issue #76.
[![translucency bug](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013731_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/wrong-translucency/unvanquished_2018-02-12_013731_000.jpg)

The problems on the blocks are fixed by #87 PR so it's likely to be the same issue:
[![weird lightmap blob](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2017-10-11_040928_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2017-10-11_040928_000.jpg)
[![weird lightmap blob](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2018-03-01_210534_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2018-03-01_210534_000.jpg)

These others issues are not fixed by #87 PR and are likely related to #76 issue.
[![weird lightmap blob](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2017-10-11_040934_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2017-10-11_040934_000.jpg)
[![weird lightmap blob](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2017-10-14_051223_000.jpg)](https://dl.illwieckz.net/b/unvanquished/bugs/weird-lightmap-blob/unvanquished_2017-10-14_051223_000.jpg)

**Comment by illwieckz** [comment]
this issue is fixed (see merged #87), other weirdness tracked in #76
